js/color-view.js

```diff
@@ -0,0 +1,56 @@
+import { html, render } from 'https://unpkg.com/lit-html@1.0.0/lit-html.js';
+
+class ColorPicker extends HTMLElement {
+    constructor() {
+        super();
+        this.render = this.render.bind(this);
+        this.setColor = this.setColor.bind(this);
+        this.applyColor = this.applyColor.bind(this);
+        this.onContextChanged = this.onContextChanged.bind(this);
+
+        fin.me.on('host-context-changed', this.onContextChanged);
+
+        // Set initial color based on current context value
+        fin.Platform.getCurrentSync().getWindowContext().then(initialContext => {
+            if (initialContext && initialContext.color) {
+                this.applyColor(initialContext.color);
+            }
+        });
+        this.render();
+    }
+
+    async render() {
+        const content = html`
+        <fieldset>
+            <p>
+                I am context-sensitive
+            </p>
+            <div>
```

Comment:
```suggestion
            <form @submit=${this.setColor}>
```
:neckbeard:  allows setting via return key

Reply:
I do like that